cookbooks/clearwater/metadata.rb

```diff
@@ -37,4 +37,4 @@
 license          "All rights reserved"
 description      "Installs/Configures clearwater"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "0.1.0"
+version          "0.0.11"
```

Comment:
I don't think you should commit this.  @graemerobertson showed me a way of ignoring local changes to this file, but I can't remember what it was.